onnxruntime/core/session/provider_bridge_ort.cc

```diff
@@ -1009,7 +1009,9 @@ struct ProviderLibrary {
     // assert(!handle_); // We should already be unloaded at this point (disabled until Python shuts down deterministically)
   }
 
-  Provider* Get() {
+  Provider* Get() ORT_TRY {
```

Comment:
>ORT_TRY [](http://example.com/codeflow?start=18&length=7)

will this work if ORT_NO_EXCEPTIONS is defined?
[https://github.com/microsoft/onnxruntime/blob/79e4ed8064cc002945f4d9bbd9dea8c234bfaf32/include/onnxruntime/core/common/common.h#L104-L108](https://github.com/microsoft/onnxruntime/blob/79e4ed8064cc002945f4d9bbd9dea8c234bfaf32/include/onnxruntime/core/common/common.h#L104-L108)
maybe better to just use try/catch directly. or put them in function body.

Reply:
Hah, now I see why we defined this. I was trying to match the other code.